packages/composite-checkout/src/components/checkout-step.js

```diff
@@ -183,24 +183,35 @@ const StepNumberInnerWrapper = styled.div`
 `;
 
 const StepNumber = styled.div`
-	background: ${getStepNumberBackgroundColor};
+	background: ${ getStepNumberBackgroundColor };
 	font-weight: normal;
 	width: 27px;
 	height: 27px;
 	line-height: 27px;
 	box-sizing: border-box;
 	text-align: center;
 	border-radius: 50%;
-	color: ${getStepNumberForegroundColor};
+	color: ${ getStepNumberForegroundColor };
 	position: absolute;
 	top: 0;
 	left: 0;
 	backface-visibility: hidden;
+	// Reason: needs to not have spaces
+	// prettier-ignore
+	@media all and (-ms-high-contrast:none), (-ms-high-contrast:active) {
+		z-index: ${ props => ( props.isComplete ? '0' : '1' ) };
+	}
 `;
 
 const StepNumberCompleted = styled( StepNumber )`
-	background: ${props => props.theme.colors.success};
+	background: ${ props => props.theme.colors.success };
 	transform: rotateY( 180deg );
+	// Reason: needs to not have spaces
```

Comment:
TIL about `prettier-ignore`! The "Reason" block is great. It might be worth including more details about why though, like "needs to not have spaces for IE11"? Or is it because of the properties themselves?

Reply:
I'll add more details but IE doesn't read the query if there are spaces in it.

Reply:
👍